versions.properties

```diff
@@ -100,6 +102,7 @@ org.rackspace:deproxy=0.21
 org.scala-lang:scala-library=2.11.7
 org.scala-lang:scala-xml=2.11.0-M4
 org.scalatest:scalatest_2.11=2.2.0
+org.python:jython=2.7.0
```

Comment:
I finally got to the bottom of the changes and thought "Yes!  I'll be able to merge this now."  Then I saw this out of order entry. :(